src/plugins/vis_type_timeseries/common/vis_schema.ts

```diff
@@ -281,5 +281,12 @@ export const visPayloadSchema = schema.object({
     min: stringRequired,
     max: stringRequired,
   }),
-  sessionId: schema.maybe(schema.string()),
+
+  searchSession: schema.maybe(
+    schema.object({
+      sessionId: schema.maybe(schema.string()),
```

Comment:
I think it would be more accurate to remove the `maybe` on the internal arguments of `searchSession` - if it's present, all fields are mandatory

Reply:
good catch improved. 
this change made payload generation on a client a bit more uglier than it was before, but I still think this makes sense.